index.js

```diff
@@ -155,10 +156,16 @@ var getType = function(value, nullable) {
     return (value.length === 0 || typeof value[0] == 'object') ? 'L' : 'SS';
   }
 
+  if (this._data.booleanSupport && typeof value == 'boolean') {
+    return "BOOL"
+  }
+
   return (typeof value == 'object') ? 'M' : Number.isFinite(value) ? 'N' : 'S';
 };
 
-var getValue = function(value, nullable) {
+Jedlik.prototype._getValue = function(value, nullable) {
+  var self = this;
```

Comment:
Do we need to store a reference to `this` here? This looks like we should still have the correct value here. If it is have the correct scope inside the forEach callback, you could just use an arrow function there.